src/lib/AnnotationModule.ts

```diff
@@ -0,0 +1,50 @@
+import { ANNOTATION_COLOR_KEY } from './constants';
+import Cache from './Cache';
+
+export enum AnnotationColor {
+    BLUE = '#0061d5',
+    GREEN_LIGHT = '#26c281',
+    GRIMACE = '#4826c2',
+    WATERMELON_RED = '#ed3757',
+    YELLORANGE = '#f5b31b',
+    YELLOW = '#ffd700',
+}
+
+type Subscription = (state: AnnotationColor) => void;
+
+export default class AnnotationModule {
+    private cache: Cache;
+
+    private subscriptions: Subscription[] = [];
+
+    constructor(cache = new Cache()) {
```

Comment:
Not sure if I should be instantiating a new cache in this constructor - I added this here because there was a type error that was showing up. I don't think we'll ever be using the new cache as there should be a cache being passed in from BaseViewer.

Reply:
Should this be an options object so it could be more easily extended in the future? Maybe could define the `type Options`

Reply:
Yep that sounds good.